cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterTwillApplication.java

```diff
@@ -117,6 +121,9 @@
                               extraClassPath);
     }
 
+    prepareHBaseDDLExecutorResources(tempDir, containerCConf,
```

Comment:
Better move this before the `if` above to group all universal resources together.